packages/compass-e2e-tests/helpers/compass.ts

```diff
@@ -994,9 +996,10 @@ export async function init(
   const { browser } = compass;
 
   // larger window for more consistent results
-  await browser.execute(() => {
-    window.resizeTo(window.screen.availWidth, window.screen.availHeight);
+  const [width, height] = await browser.execute(() => {
+    return [window.screen.availWidth, window.screen.availHeight];
```

Comment:
firefox ignores window.resizeTo() unless it is a window opened using window.open(). But luckily there's browser.setWindowSize()